tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1117,9 +1117,28 @@ public function testAlterTableColumnAdd()
 
         $table->addColumn('string_col', 'string', ['default' => '']);
         $table->addColumn('string_col_2', 'string', ['null' => true]);
+        $table->addColumn('string_col_3', 'string');
+        $table->addTimestamps();
         $table->save();
-        $this->assertTrue($this->adapter->hasColumn('table1', 'string_col'));
-        $this->assertTrue($this->adapter->hasColumn('table1', 'string_col_2'));
+
+        $columns = $this->adapter->getColumns('table1');
+        $expected = [
+            ['name' => 'id', 'type' => 'integer', 'default' => null, 'null' => false],
+            ['name' => 'string_col', 'type' => 'string', 'default' => '', 'null' => false],
+            ['name' => 'string_col_2', 'type' => 'string', 'default' => '', 'null' => true],
+            ['name' => 'string_col_3', 'type' => 'string', 'default' => '', 'null' => false],
+            ['name' => 'created_at', 'type' => 'timestamp', 'default' => 'CURRENT_TIMESTAMP', 'null' => false],
+            ['name' => 'updated_at', 'type' => 'timestamp', 'default' => null, 'null' => true],
+        ];
+
+        $this->assertEquals(count($expected), count($columns));
+        $columnCount = count($columns);
+        for ($i = 0; $i < $columnCount; $i++) {
+            $this->assertEquals($expected[$i]['name'], $columns[$i]->getName(), "Wrong name for {$expected[$i]['name']}");
+            $this->assertEquals($expected[$i]['type'], $columns[$i]->getType(), "Wrong type for {$expected[$i]['name']}");
+            $this->assertEquals($expected[$i]['default'], $columns[$i]->getDefault(), "Wrong default for {$expected[$i]['name']}");
```

Comment:
Can we use assertSame() please. The false positive comparison of null vs false etc can be quite a problem.